physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -747,6 +769,43 @@
   type = real
   kind = kind_phys
   intent = inout
+[mpicomm]
+  standard_name = mpi_communicator
+  long_name = MPI communicator
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[mpirank]
+  standard_name = mpi_rank
+  long_name = current MPI-rank
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[mpiroot]
+  standard_name = mpi_root
+  long_name = master MPI-rank
+  units = index
+  dimensions = ()
+  type = integer
+  intent = in
+[uspdavg]
+  standard_name = bl_averaged_wind_speed
+  long_name = average wind speed within the boundary layer
+  units = none
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = inout
+[hpbl_thetav]
+  standard_name = pbl_height_thetav
```

Comment:
There is a CF variable (that is also used in the UFS) that is atmosphere_boundary_layer_thickness that is used for the PBL height. You could use `atmosphere_boundary_layer_thickness_from_modified_parcel`. Yes, it's long, but if you need to differentiate this from the normal PBL height, the standard name should make it obvious what is different about it.

Reply:
@grantfirl Thanks for your information and suggestion. atmosphere_boundary_layer_thickness_from_modified_parcel will be updated as stand name.